sdks/java/extensions/sql/build.gradle

```diff
@@ -73,7 +73,13 @@ dependencies {
   compile "com.alibaba:fastjson:1.2.69"
   compile "org.codehaus.janino:janino:3.0.11"
   compile "org.codehaus.janino:commons-compiler:3.0.11"
-  provided "org.checkerframework:checker-qual:3.4.1"
+  compile library.java.jackson_core
+  compile library.java.vendored_guava_26_0_jre
+  compile "org.mongodb:mongo-java-driver:3.9.1"
```

Comment:
This is a different version than the one set in `:sdks:java:io:mongodb`, which is where this transitive dependency comes from. We should probably make this a common library in BeamModulePlugin so they can use the same value.

https://github.com/apache/beam/blob/96b4a5813e42733deca57b6198eaf995b5ac3fc2/sdks/java/io/mongodb/build.gradle#L32

Reply:
@ibzib I've created PR 13910 to create this mongo-java-driver variable under library.java. If you can merge that one, I can proceed with using that same variable here.